external/deep-object-reid/tools/ote_sample.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
Does it make any sense to still keep sample when it's located side by side with CLI?

Reply:
probably not, @sovrasov ?

Reply:
@sovrasov please respond, will remove it later if it is needed

Reply:
Yes, we can remove it